src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt

```diff
@@ -36,7 +36,7 @@ class RsFileStub : PsiFileStubImpl<RsFile> {
 
     object Type : IStubFileElementType<RsFileStub>(RsLanguage) {
         // Bump this number if Stub structure changes
-        override fun getStubVersion(): Int = 180
+        override fun getStubVersion(): Int = 181
```

Comment:
Update it please according to the latest value in master

Reply:
Fixed.